Makefile

```diff
@@ -1,20 +1,18 @@
 include packaging/Makefile.packaging
 
 .PHONY: depend clean test build tarball
-.DEFAULT: build
```

Comment:
This never did anything. Whoever wrote it probably meant to set `build` as the default target, but [that isn't what `.DEFAULT` does](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html). In fact, the prerequisites of `.DEFAULT` are ignored except when literally running `make .DEFAULT` from a command-line.

There's a special `.DEFAULT_GOAL` variable that would set the default target, but it's more common to just list the default target first.